docker-compose.yaml

```diff
@@ -40,5 +40,8 @@ services:
       POSTGRES_DB: timelessdb_test
       POSTGRES_USER: timeless_user
       POSTGRES_PASSWORD: timeless_pwd
+  redis:
```

Comment:
@paulodamaso 
```
  redis:
    image: 'redis:3.2'
    ports:
      - '6379:6379'
```

Reply:
@paulodamaso also pls add `redis` in `depends_on` section for `app` service:
```yaml
app:
    ....
    depends_on:
      - db
      - redis
```

Reply:
@vryazanov thanks for the help